packages/create-sitecore-jss/src/templates/vue/sitecore/definitions/templates/Styleguide-EditFrame-ListItem-Template.sitecore.js

```diff
@@ -0,0 +1,12 @@
+import { CommonFieldTypes, Manifest } from '@sitecore-jss/sitecore-jss-dev-tools';
```

Comment:
As I remember, lint warning will appear in this file, since Manifest type is unused here, so eslint comment should be reverted back